pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -130,11 +130,9 @@ func (c *cephStatusChecker) checkStatus() {
 		logger.Errorf("failed to query cluster status in namespace %q. %v", c.clusterInfo.Namespace, err)
 	}
 
-	if status.Health.Status != "HEALTH_OK" {
```

Comment:
if there is no osd or mon pod on a failed node, the health can remain in `HEALTH_OK`, so removed the check

Reply:
Will removing this result in unnecessary thrashing of the k8s apiserver? Is there a case where we need to clean up stuck pods when health is `HEALTH_OK`?

Reply:
Added it back.
My thought on removing this check was https://github.com/rook/rook/pull/7266#discussion_r578699031 but its not the case.